openapi.yaml

```diff
@@ -5325,6 +5325,76 @@ paths:
       - lang: CLI
         source: >
           linode-cli linodes upgrade 123
+  /linode/instances/{linodeId}/password:
+    parameters:
+    - name: linodeId
+      in: path
+      description: ID of the Linode for which to reset it's root password.
+      required: true
+      schema:
+        type: integer
+    x-linode-cli-command: linodes
+    post:
+      x-linode-grant: read_write
+      summary: Reset Linode Root Password
+      description: >
+        Resets the root password for this Linode.
+
+
+
+        **Note:** If your Linode has more than one disk (not counting its swap disk), use the
+        [Reset Disk Root Password](/api/v4/linode-instances-linode-id-disks-disk-id-password/#post) endpoint to update a specific disk's root password.
+
+
+        A `password_reset` event is generated when a root password reset is successful.
```

Comment:
This should probably note that the Linode must be powered off for a password reset to succeed.

Reply:
@Dorthu I added a note about this: https://github.com/linode/linode-api-docs/pull/319/commits/cf786498244fcaebb6e692b0462353de4e9b507b#diff-fe030a7c1568b3decf599edf399be7f4R5343